iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/integration/test/util/IcebergRESTServerManagerForDeploy.java

```diff
@@ -51,12 +55,39 @@ public Optional<Future<?>> doStartIcebergRESTServer() throws Exception {
         SQLITE_DRIVER_DOWNLOAD_URL,
         Paths.get(icebergRESTServerHome.toString(), "iceberg-rest-server", "libs").toString());
 
-    String cmd = String.format("%s/bin/%s start", icebergRESTServerHome.toString(), SCRIPT_NAME);
-    CommandExecutor.executeCommandLocalHost(
-        cmd,
-        false,
-        ProcessData.TypesOfData.OUTPUT,
-        ImmutableMap.of("GRAVITINO_HOME", icebergRESTServerHome.toString()));
+    String gravitinoRestStartShell = icebergRESTServerHome.toString() + "/bin/" + SCRIPT_NAME;
```

Comment:
remove this?

Reply:
Do you mean we need to remove this code block? I'm afraid not, removing it will make Iceberg REST server start failed when using Kerberos Hive backend.